requirements-dev.txt

```diff
@@ -14,6 +16,7 @@ pytest
 pytest-cases
 pytest-cov
 python-dotenv
+pywin32; sys_platform == 'win32'
```

Comment:
Huh, was the VM you were using win32? Didn't even know 32-bit stuff was still around.

Reply:
😆 I don't think that it was, but there's a bunch for historical reasons I think the the os level api on windows is often referred to as "win32"